build.conf

```diff
@@ -64,15 +61,14 @@ BUILD_SQLITE="yes"
 PIP_INDEX_URL="https://bin.chevah.com:20443/pypi/simple"
 # Array of default Python modules installed using pip.
 PIP_LIBRARIES=(\
-    cffi=="$CFFI_VERSION" \
-    bcrypt=="$BCRYPT_VERSION" \
     psutil=="$PSUTIL_VERSION" \
```

Comment:
I think that we can also remove psutil for pythia.

Reply:
Unfortunately, there is no musl wheel for `psutil`.